docs/runtime_usage.md

```diff
@@ -220,5 +220,25 @@ val dyn = js("""{s:"foo", d:{a:42}}""")
 val parsed = DynamicObjectParser().parse<DataWrapper>(dyn)
 parsed == DataWrapper("foo", Data(42)) // true
 ```
-
 > Parser does not support kotlin maps with keys other than `String`.
+
+ ### Dynamic object serializer 
+ 
+ Allows you to convert kotlin data structures into their dynamic JS representation.
+
+```kotlin
+
+@Serializable
+data class Data(val a: Int)
+
+@Serializable
+open class DataWrapper(open val s: String, val d: Data?)
```

Comment:
I thought it was important to show how we are unaffected by name mangling here since it is one of the use cases for DynamicObjectSerializer. 
We could also just mention it in the text above and leave out the `open val` from the example if you prefer.